pkgs/applications/video/kodi/addons/jellycon/default.nix

```diff
@@ -29,7 +29,7 @@ buildKodiAddon rec {
   '';
 
   postInstall = ''
-    mv /build/source/addon.xml $out${addonDir}/${namespace}/
+    cp -v addon.xml $out${addonDir}/$namespace/
```

Comment:
`install`?

Reply:
due to `-t` i find `install` way harder to read than `cp` whose order of arguments are consistent, and the file permissions are fine as is.

Reply:
do you consider this a blocker or are we good to merge?

Reply:
Not at all.